src/components/CareCards/index.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import CareCard from '../CareCard/index';
```

Comment:
When you import a component you can avoid adding the `/index` javascript detect it automatically.
I look at this in all components where you include other components.

Reply:
all changes done